parsec/backend/memory/invite.py

```diff
@@ -173,6 +174,14 @@ async def new_for_user(
         claimer_email: str,
         created_on: Optional[Pendulum] = None,
     ) -> UserInvitation:
+        """
+        Raise: InvitationAlreadyMemberError
+        """
+        org = self._user_component._organizations[organization_id]
+        for _, user in org.users.items():
+            is_revoked = user.revoked_on and user.revoked_on <= pendulum_now()
```

Comment:
don't we already have a `user.is_revoked` property to do this check ? (I can't retrieve it in the codebase but I thought I saw something like this... @AntoineDupre any idea ?)

Reply:
@touilleMan I don't think so. I found that we test if a user is revoked in different places in the same way (like [there](https://github.com/Scille/parsec-cloud/blob/master/parsec/backend/handshake.py#L112) )

It can be nice to added it as there is an `is_expired` for the organization:
```python
@attr.s(slots=True, frozen=True, auto_attribs=True)
class User:
 ...
    @property
    def is_revoked(self):
        return user.revoked_on and user.revoked_on <= pendulum_now()
```

Reply:
@AntoineDupre yep it would be good to add this then

btw I think a method is better than a property here given it result depends of side effects (so doing `is_revoked = user.is_revoked [...] if is_revoked [...] if is_revoked [...]` is not equivalent to `if user.is_revoked [...] if user.is_revoked [...]`)